packages/types/src/permission.ts

```diff
@@ -0,0 +1,14 @@
+import type { ClerkResource } from './resource';
+
+/**
+ * @experimental
+ */
+export interface PermissionResource extends ClerkResource {
+  id: string;
+  key: string;
+  name: string;
+  type: 'system';
```

Comment:
❓ Do we only support `system` for now?

Reply:
just checked, seems like we also support `user`, i'm adding that now